opscenter/6.1/Dockerfile

```diff
@@ -28,20 +36,24 @@ RUN apt-get update -qq \
     && rm -rf /var/lib/{apt,dpkg,cache,log}/
 
 
-# Set up configuration variables
-ENV OPSCENTER_HOME=/opt/opscenter
-
 # Copy any special files into the image
 COPY files /
 
-# Download DataStax Studio if needed
+# Download DataStax Opscenter if needed
 RUN set -x \
     && if test ! -e /$OPSCENTER_TARBALL; then wget -q -O "/$OPSCENTER_TARBALL" "$OPSCENTER_DOWNLOAD_URL"; fi \
 # Unpack the tar ball
     && mkdir -p "$OPSCENTER_HOME" \
     && tar -C "$OPSCENTER_HOME"  --strip-components=1 -xzf /$OPSCENTER_TARBALL \
     && rm /$OPSCENTER_TARBALL \
-    && mkdir /var/lib/opscenter
+    && chown -R opscenter:opscenter ${OPSCENTER_HOME} \
+
+# Create volume folders
+    && (for dir in /var/lib/opscenter /config ; do \
+        mkdir -p $dir && chown -R opscenter:opscenter $dir && chmod 777 $dir ; \
+    done )
+
+USER opscenter
```

Comment:
you can move `USER` statement before line 42 and with that all following commands will be executed on behalf of that user. Hence, we could drop `chown` and `chmod` commands.

Reply:
Cannot move USER before line 42, since that is before the tarball download which is downloaded to `/`